compiler/util/clangUtil.cpp

```diff
@@ -994,13 +1004,16 @@ void finishCodegenLLVM() {
   info->FPM_postgen = NULL;
 
   // Now finish any Clang code generation.
-  cleanupClang(info);
+  finishClang(info);
+
+  if(debug_info)debug_info->finalize();
```

Comment:
OK, so we need to avoid calling cleanupClang before debug_info->finalize(), but it would be nice to call cleanupClang() eventually. Could it be called right here after debug_info->finalize() ?